app/views/admin/orders/edit.html.erb

```diff
@@ -0,0 +1,70 @@
+    <%= t("admin.orders_admin.edit.title", order_id: @order.id) %> 
+  </h1>
+
+  <%= form_with(model: [:admin, @order], local: true) do |f| %>
+    <div class="bg-white p-8 rounded-lg shadow-md mb-8">
+      <h2 class="text-2xl font-semibold text-gray-700 mb-6 border-b pb-2">
+        <%= t("admin.orders_admin.edit.info_title") %> 
+      </h2>
+      <div class="grid grid-cols-1 md:grid-cols-2 gap-6">
+        <div>
+          <div class="mb-4">
+            <%= f.label 'address[receiver_name]', t("admin.orders_admin.edit.receiver_name"), class: "font-medium mb-2" %> 
+            <%= f.text_field 'address[receiver_name]', value: @order.address.receiver_name, class: "block mt-2 w-full border border-gray-300 rounded-lg px-3 py-2", disabled: @order.cancelled? %>
+          </div>
+          <div class="mb-4">
+            <%= f.label 'address[place]', t("admin.orders_admin.edit.address"), class: "font-medium mb-2" %> 
+            <%= f.text_field 'address[place]', value: @order.address.place, class: "block mt-2 w-full border border-gray-300 rounded-lg px-3 py-2", disabled: @order.cancelled? %>
+          </div>
+          <div class="mb-4">
+            <%= f.label 'address[phone]', t("admin.orders_admin.edit.phone"), class: "font-medium mb-2" %> 
+            <%= f.text_field 'address[phone]', value: @order.address.phone, class: "block mt-2 w-full border border-gray-300 rounded-lg px-3 py-2", disabled: @order.cancelled? %>
+          </div>
+        </div>
+        <div>
+          <div class="mb-6">
+            <%= f.label :payment_method, t("admin.orders_admin.edit.payment_method"), class: "font-medium text-gray-700" %> 
+            <%= f.select :payment_method, 
+                  options_for_select([ 
+                    [t("admin.orders_admin.edit.payment_methods.cash_on_delivery"), 'cash_on_delivery'], 
+                    [t("admin.orders_admin.edit.payment_methods.credit_card"), 'credit_card']
+                  ], @order.payment_method),
+                  {}, 
+                  class: "block mt-2 w-full border border-gray-300 rounded-lg px-4 py-3 focus:ring-primary focus:border-primary", 
+                  disabled: @order.cancelled? %>
+          </div>
+          <div class="mb-6">
+            <%= f.label :status, t("admin.orders_admin.edit.status"), class: "font-medium text-gray-700" %> 
+            <% order_statuses = [
+              { label: t("admin.orders_admin.edit.statuses.pending"), value: 'pending' },
+              { label: t("admin.orders_admin.edit.statuses.confirmed"), value: 'confirmed' },
+              { label: t("admin.orders_admin.edit.statuses.preparing"), value: 'preparing' },
+              { label: t("admin.orders_admin.edit.statuses.delivering"), value: 'delivering' },
+              { label: t("admin.orders_admin.edit.statuses.delivered"), value: 'delivered' },
+              { label: t("admin.orders_admin.edit.statuses.cancelled"), value: 'cancelled' }
+            ] %>
```

Comment:
Đưa vào trong helper nhé
Sử dụng Order.status để lấy ra toàn bộ các status
rồi đưa vào vòng lặp để tạo mảng